cfg/vic20-asm.cfg

```diff
@@ -7,7 +7,7 @@ SYMBOLS {
 MEMORY {
     ZP:       file = "", start = $0002,  size = $001A, define = yes;
     LOADADDR: file = %O, start = %S - 2, size = $0002;
-    MAIN:     file = %O, start = %S,     size = $0DF3 - %S;
+    MAIN:     file = %O, start = %S,     size = $1E00 - %S;
```

Comment:
It would be great if you could add some comment to this file that
- tell what vic20 memory config it is meant to be used with (it should be "unexpanded")
- how to modify this for other configs (+8k, full)